lib/page/home/home_page.dart

```diff
@@ -116,16 +117,20 @@ class _HomePageState extends ConsumerState<HomePage> {
                 child: ListView(),
               ),
             SafeArea(
-              child: HomeHeaderWidget(
-                currentDate:
-                    ref.read(homeViewModelProvider.notifier).getCurrentDate(),
-                userAvatarUrl: ref.watch(_userStreamProvider).value?.avatarUrl,
+              child: Column(
+                children: [
+                  HomeHeaderWidget(
+                    currentDate: ref
+                        .read(homeViewModelProvider.notifier)
+                        .getCurrentDate(),
+                    userAvatarUrl:
+                        ref.watch(_userStreamProvider).value?.avatarUrl,
+                  ),
+                  if (shouldShowLoading)
+                    LoadingIndicatorWidget(shouldIgnoreOtherGestures: false)
+                ],
```

Comment:
Are you sure you would like to show the new Loading progress on the bottom of the Home screen by using Column?

Please provide POW for this screen update in PR's description 🙏

Reply:
Actually, it is under the "Today" text 🙏🏼 You can check out the newly added POW above

Reply:
@chornerman Yes, but why do we put it under the Home Header? Should we put it in the center screen instead (I mean using Stack)?

Reply:
@luongvo I try to follow this design 🙏🏼 

https://user-images.githubusercontent.com/13492460/209612877-8a86b05f-ccfa-4ea8-9bed-2f1e5588d179.mov

Reply:
@chornerman oh, I don't see this anim before, good to know 👍